server/main.py

```diff
@@ -153,6 +185,21 @@ async def room(websocket: WebSocket, room_name: str) -> None:
     """
     client = Client(websocket)
     await manager.connect(client, room_name)
+    initial_data = ConnectionData(**await client.receive())
```

Comment:
This raises a `WebSocketDisconnect` exception if the client disconnects before it sends any data.

Reply:
What are you doing to cause that? Can't reproduce.

Reply:
Just disconnect before sending anything. The error will appear on the server.

Reply:
Not getting anything?
![image](https://user-images.githubusercontent.com/68163698/180668435-cf57fccd-6e13-467d-a465-cf741ed1e3ec.png)

Reply:
> Not getting anything? ![image](https://user-images.githubusercontent.com/68163698/180668435-cf57fccd-6e13-467d-a465-cf741ed1e3ec.png)

It's because you are sending the `connect` event. This is what I get:
![image](https://user-images.githubusercontent.com/72104527/180668641-09c8366a-93ee-4aa7-aa23-f9f575c3541a.png)

Reply:
I see. Will have a look

Reply:
Works now.